skip failover on interrupted thread (#1284)
aws/aws-advanced-jdbc-wrapper: docs/using-the-jdbc-driver/using-plugins/UsingTheFailover2Plugin.md

> | `skipFailoverOnInterruptedThread`     | Boolean |                                        No                                        | Allows to avoid starting failover if the current thread is interrupted. Connection may be left in invalid state and it should be disaposed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | `false`                                                                                                                                                                                             |

Reviewer: ```suggestion
| `skipFailoverOnInterruptedThread`     | Boolean |                                        No                                        | Enable to skip failover if the current thread is interrupted. This may leave the Connection in an invalid state so the Connection should be disposed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | `false`                                                                                                                                                                                             |
```
